### PR TITLE
Node18 kubevirt plugin

### PR DESCRIPTION
**Pull request by upalatucci**
Switch to node18 builder.

`tectonic-console-builder-v24` is Node 14

### CODE REVIEW

File: ci-operator/config/kubevirt-ui/kubevirt-plugin/kubevirt-ui-kubevirt-plugin-main.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: console-plugin-test-cypress
+    tag: tectonic-console-builder-28
```
Comment (vikram-raj) [inline ci-operator/config/kubevirt-ui/kubevirt-plugin/kubevirt-ui-kubevirt-plugin-main.yaml:5]: Here this tag should be `tectonic-console-builder-v28`. This is what is mentioned in the release.yaml.

Comment (upalatucci) [reply ci-operator/config/kubevirt-ui/kubevirt-plugin/kubevirt-ui-kubevirt-plugin-main.yaml]: thanks man !! You've just saved my day :D

**Comment by upalatucci** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Comment by metalice** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Comment by metalice** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Comment by upalatucci** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Comment by upalatucci** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Comment by upalatucci** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Comment by metalice** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Comment by metalice** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Comment by upalatucci** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Comment by upalatucci** [comment]
/pj-rehearse pull-ci-kubevirt-ui-kubevirt-plugin-main-kubevirt-e2e-aws

**Pull request merged**